src/Particle.js

```diff
@@ -2099,6 +2103,29 @@ class Particle {
 		});
 	}
 
+	/**
+	 * Set default auth token that will be used in each method if `auth` is not provided
+	 * @param  {String} auth A Particle access token
+	 * @returns {undefined}
+	 */
+	setDefaultAuth(auth){
+		this._defaultAuth = auth;
+	}
+	/**
+	 * Return provided token if truthy else use default auth if truthy else undefined
+	 * @param {*} auth Optional auth token or undefined
+	 * @private
+	 * @returns {String|undefined} a Particle auth token or undefined
+	 */
+	_getEffectiveAuth(auth) {
+		if (auth) {
+			return auth;
+		} else if (this._defaultAuth) {
+			return this._defaultAuth;
+		} else {
+			return undefined;
+		}
+	}
```

Comment:
also a nit, not blocking: this could be simplified to:

```suggestion
		return auth || this._defaultAuth
	}
```